uui/components/datePickers/__tests__/RangeDatePickerBody.test.tsx

```diff
@@ -173,14 +192,80 @@ describe('RangeDatePickerBody', () => {
         const [oct13] = useCase2.result.queryAllByText('13');
         const [oct14] = useCase2.result.queryAllByText('14');
 
-        function parentElemContainsClasses(elem: HTMLElement, classesArr: string[]) {
-            // @ts-ignore
-            const actualList = [...elem.parentElement.classList];
-            return classesArr.every((c: string) => actualList.indexOf(c) !== -1);
-        }
+        expect(parentElemContainsClasses(oct12, [uuiRangeDatePickerBody.firstDayInRangeWrapper, uuiDaySelection.selectedDay])).toBeTruthy();
```

Comment:
It's not a good practice to rely on css classes in tests. Probably, we can add for this nodes some html attributes(may be from a11y) and then use them in tests except using classnames?

Reply:
discussed offline